src/scrabbleScore.js

```diff
@@ -24,17 +23,22 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
-  // This is your job. :)
+  let score = 0;
+  let wordArray = word.toLowerCase().split('');
+
+  for(let letter of wordArray){
```

Comment:
`for ... of` lets you iterate over strings one letter at a time, so no need to convert it to an array first.

```javascript
for (let letter of words.toLowerCase()) {
  // ...
}
```